user-guide/Advanced_Functionality/Databases/Indexing_database/Profile_migration_to_elastic.md

```diff
@@ -51,7 +51,10 @@ To migrate the profiles, you will need to use the SLNetClientTest tool. Note tha
 
    > [!NOTE]
    > The migration should not take more than a few minutes. During testing, migrating a *Profiles.xml* file of 100 MB in a system with a local Elasticsearch database took about 4 minutes.
-   > In the MigrationStatus table in SLNetClientTest tool, the profile instances, definitions and parameters rows will reflect the progress of the migration. 
+
+1. Check whether in the *MigrationStatus* table, the migration of profile instances, definitions, and parameters has the status completed, and there are no error messages.
+
+   If this is the case, the migration is complete.
```

Comment:
What should users do in case there are error messages? It might be helpful if we added this here.

Reply:
This is mentioned in the 'Troubleshooting' section just beneath this text. Is it ok like this or should it be mentioned in this bit of text as well?

Reply:
I'll add a link here as well then 😊